.github/workflows/tests.yaml

```diff
@@ -5,62 +5,65 @@ env:
   PGUSER: postgres
   PGDATABASE: ccbc
 
-on: 
+on:
   workflow_call:
     secrets:
-      SONAR_TOKEN: 
+      SONAR_TOKEN:
         required: true
 
-
 jobs:
   install-test-env:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
+      - run: make install_dev_tools
+      - run: make stop_pg # cleanly stop the database, otherwise it will complain in the subsequent jobs
```

Comment:
doesn't `install_dev_tools` depend on `stop_pg`? Is it restarting it again?

Reply:
yes, the pgtap install starts pg

Reply:
should it maybe not?

Reply:
it needs to start pg to install the extension and run its tests, but it should probably stop it after...